lib/excel.py

```diff
@@ -23,5 +23,5 @@ def empty_cell(cell):
     def cell_value(cell):
         """Returns the string value of an excel spreadsheet cell"""
         if (cell.value).__class__.__name__ == 'unicode':
-            return (cell.value).encode().strip()
+            return (str(cell.value).encode()).strip()
```

Comment:
@goelakash, it seems like striping the after str() did the trick.
you could change whatever you feel is worth refactoring.

@ethanwhite, this is failing because of codecov,